NEWS

```diff
@@ -11,6 +11,15 @@
 ### Features:
 ### Bugfixes:
 
+## 1.13.0-rc2 (July 24)
```

Comment:
since it's 1.13 release need to update the 1.13.0 entry and set date to today

Reply:
then maybe update 1.13.0 entry to 1.13.0-rc1 and add new one 1.13.0-rc2 (today)?